src/wxflow/template.py

```diff
@@ -189,3 +189,13 @@ def is_single_type_or_string(s):
         return True
     else:
         return False
+
+
+def replace_tmpl(string, tmpl_dict):
+    """
+        Replace substrings of input string using  input dictionary.
+    """
+
+    for key in tmpl_dict:
+        string = string.replace(key, tmpl_dict[key])
+    return string
```

Comment:
Since this is used as a standalone filter in jinja2, can this be made a private method in `jinja2.py`? 

Also, please add a test for this function.